torch/csrc/jit/codegen/cuda/fusion.h

```diff
@@ -153,7 +154,8 @@ struct TORCH_CUDA_API Fusion : public IRInputOutput {
    */
   std::vector<Expr*> exprs(
       bool from_outputs_only = false,
-      bool breadth_first = false);
+      bool breadth_first = false,
```

Comment:
we could remove breadth_first - there's no implementation or use for it. If we ever want it it's easy to add it back.

@csarofeen , what do you think?

Reply:
sure, sounds good.

Reply:
I'll take care of it later as a separate PR (#130)